paramfetch.go

```diff
@@ -174,7 +174,8 @@ func doFetch(out string, info paramFile) error {
 	}
 	defer resp.Body.Close()
 
-	bar := pb.New64(resp.ContentLength)
+	bar := pb.New64(fStat.Size() + resp.ContentLength)
+	bar.Set64(fStat.Size())
```

Comment:
This does not affect speed measurements because `ProgressBar.startValue` is taken from `ProgressBar.current` at `ProgressBar.Start()`.